### DISCUSSION TITLE
Regular style icons included in free package but usage is pro only?

### DISCUSSION THREAD

**Discussion by skrysm** [FortAwesome/Font-Awesome#19410] [Q&A] [answered]
I'm a little confused about whether I can use the "regular" style icons with a free license or not.

When downloading the "Free for Web" archive (from <https://fontawesome.com/download>), the archive contains the `fa-regular-400` font.

But on the [Host Yourself - Web Fonts](https://fontawesome.com/docs/web/setup/host-yourself/webfonts) page it says that the "regular" font is "Pro only".

If it's "Pro only", why include it in the free package in the first place? Also: It seems kind of dangerous to include it in the free package as in "you risk a license violation without noticing".

Or am I missing something here?

**Comment by tagliala** [comment, marked as answer]
Hi,

a subset of regular icons (163 out of 3000+ at the moment I'm writing) is available in Font Awesome Free: https://fontawesome.com/search?o=r&m=free&s=regular

Feel free to use them

The reason is historical. Icons known as "outline" in FA4 were moved to the regular set, and to keep backward compatibility and because some of them are useful for status switching (e.g.: `fa-solid heart <-> fa-regular heart`), they are in the free set

Hope this clarifies

**Reply by skrysm** [reply]
Yes, thank you.
